### PR TITLE
Let's try this quest thing out

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary
Let's try this quest thing out

### CODE REVIEW

File: .github/workflows/quest.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        description: "The issue number to manually test"
+        required: true
+
+jobs:
+  import:
+    if: github.event.label.name == join(fromJson('[":", "world_map", ":", " reQUEST"]'), '') || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
```
Comment (BillWagner) [inline .github/workflows/quest.yml:22]: This shouldn't need to write pull requests.

**Review by BillWagner — APPROVED** [review]
One small comment, and then let's try it.

I'm not seeing how the command line args get to the app, but that might be just my ignorance.

File: .github/workflows/quest.yml (hunk 1 above)
Comment (IEvangelist) [inline .github/workflows/quest.yml:22]:
```suggestion
      issues: write
```

Comment (IEvangelist) [inline .github/workflows/quest.yml:22]:
```suggestion
      issues: write
```

**Pull request merged**